packages/backend/src/graphql/mutations/tiles/delete-table.ts

```diff
@@ -0,0 +1,27 @@
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    id: string
+  }
+}
+
+const deleteTable = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  const table = await context.currentUser
+    .$relatedQuery('tables')
+    .findOne({
+      id: params.input.id,
+    })
+    .throwIfNotFound()
+
+  await table.$relatedQuery('columns').delete()
```

Comment:
nit: should we have transaction here? In theory not needed because we delete columns first, so worst case we just have empty table if there are any errors.

But feels like all-or-nothing would make more sense from user POV - if they get error while while deleting a table, they won't expect all their columns to be suddenly gone but the table is still accessible.